idaes/generic_models/flowsheets/methanol_flowsheet.py

```diff
@@ -11,386 +11,12 @@
 # at the URL "https://github.com/IDAES/idaes-pse".
 ##############################################################################
 """
-Task: IDAES Support for ARPE-E Differentiate
-Scenario: Methanol Synthesis From Syngas
-Author: B. Paul and M. Zamarripa
+Deprecation path for renamed module.
```

Comment:
This file is not needed - the methanol flowsheet is a new addition to the repo so it does not need a deprecation path.